simvue_fds/connector.py

```diff
@@ -520,6 +678,10 @@ def _post_simulation(self):
                         continue
                     self.save_file(file, "output")
 
+        # Then wait for slice parser to finish
+        while self._parsing:
+            time.sleep(10)
```

Comment:
Yh this feels arbitrary, for the tests I wait for N seconds but check at 1s intervals so it can continue ASAP. Not sure if that is a bad idea or not so just take this as a comment for discussion.